server/src/main/java/com/paypal/selion/grid/matchers/SeLionSauceCapabilityMatcher.java

```diff
@@ -18,18 +18,19 @@
 import java.util.Map;
 
 import org.openqa.grid.internal.utils.CapabilityMatcher;
-import org.openqa.grid.internal.utils.DefaultCapabilityMatcher;
+
+import com.paypal.selion.proxy.SeLionSauceProxy;
 
 /**
- * This is a {@link CapabilityMatcher} based extension of {@link DefaultCapabilityMatcher} that nullifies the 
- * capabilities of the internal {@link DefaultCapabilityMatcher} which resides in the Grid.
- * This is essentially done to ensure that all of the {@link CapabilityMatcher} capabilities are delegated
- * to the one that dwells in the Sauce Labs cloud.
+ * Used in conjunction with a grid that has a {@link SeLionSauceProxy} connected. <br>
+ * <br>
+ * A simple {@link CapabilityMatcher} which delegates matches against node capabilities to the Sauce Labs cloud when
+ * {@link SeLionSauceProxy} forwards the session.
  * 
  */
-public class SeLionSauceCapabilityMatcher extends DefaultCapabilityMatcher {
-    @Override
-    public boolean matches(Map<String, Object> nodeCapability, Map<String, Object> requestedCapability) {
+public class SeLionSauceCapabilityMatcher implements CapabilityMatcher {
```

Comment:
I ask only because I still have a vision to unify our proxies at some point.   If for no other reason then to simplify auto provisioning of grid in the future.